### PR TITLE
Add Padding

### PR DESCRIPTION
**Pull request by BEAdi**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-44110

## Description
Add padding to decoding a base 64 image to binary, to avoid "Incorrect padding" error.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ApiModules/ReleaseNotes/2_2_30.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### MicrosoftGraphMailApiModule
+
+Fixed an issue where the ***send-mail***, ***reply-mail***, ***get-attachments*** and ***fetch-incidents*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/ApiModules/ReleaseNotes/2_2_30.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
- ***get-attachments***
- ***fetch-incidents*** 
```

File: Packs/Gmail/ReleaseNotes/1_3_29.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gmail
+
+Fixed an issue where the ***send-mail*** and ***reply-mail*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/Gmail/ReleaseNotes/1_3_29.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
```

File: Packs/GmailSingleUser/ReleaseNotes/1_4_9.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Gmail Single User
+
+Fixed an issue where the ***send-mail*** and ***reply-mail*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/GmailSingleUser/ReleaseNotes/1_4_9.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
```

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_5_18.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS O365
+
+Fixed an issue where the ***send-mail*** and ***reply-mail*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_5_18.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
```

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Microsoft Graph Mail Single User
+
+Fixed an issue where the ***send-mail***, ***reply-mail***, ***get-attachments*** and ***fetch-incidents*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
- ***get-attachments***
- ***fetch-incidents*** 
```

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+##### O365 Outlook Mail (Using Graph API)
+
+Fixed an issue where the ***send-mail***, ***reply-mail***, ***get-attachments*** and ***fetch-incidents*** commands failed to decode the **htmlBody** argument if it was provided with no padding.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md:10]:
```suggestion
Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
- ***send-mail***
- ***reply-mail***
- ***get-attachments***
- ***fetch-incidents*** 
```

**Comment by ShirleyDenkberg** [comment]
@tcarmeli1 Doc review completed.

File: Packs/ApiModules/ReleaseNotes/2_2_30.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### MicrosoftGraphMailApiModule
+
+Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
```
Comment (tcarmeli1) [inline Packs/ApiModules/ReleaseNotes/2_2_30.md:6]: You fixed also comments that do not use the htmlBody such as file_result_creator inside the create_attachment command (also would suggest changing the get-attachments to the real name), maybe i would change the release notes to :
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the htmlBody when provided without padding:
- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 
wdyt?

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+
+##### Microsoft Graph Mail Single User
+
+Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
```
Comment (tcarmeli1) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md:6]: Same as above:
You fixed also comments that do not use the htmlBody such as file_result_creator inside the create_attachment command (also would suggest changing the get-attachments to the real name), maybe i would change the release notes to :
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the htmlBody when provided without padding:

send-mail
reply-mail
msgraph-mail-get-attachment
fetch-incidents
wdyt?

**Review by tcarmeli1 — CHANGES_REQUESTED** [review]
Great Work! :)

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+- ***reply-mail***
+- ***msgraph-mail-get-attachment***
+- ***fetch-incidents*** 
+
+##### O365 Outlook Mail (Using Graph API)
+
+Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
+- ***send-mail***
+- ***reply-mail***
+- ***get-attachments***
```
Comment (tcarmeli1) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md:17]:
```suggestion
- ***msgraph-mail-get-attachment***
```

**Review by tcarmeli1 — CHANGES_REQUESTED** [review]

File: Packs/ApiModules/ReleaseNotes/2_2_30.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### MicrosoftGraphMailApiModule
+
+Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html נody when provided without padding:
```
Comment (BEAdi) [inline Packs/ApiModules/ReleaseNotes/2_2_30.md:6]:
```suggestion
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html body when provided without padding:
```

File: Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+- ***fetch-incidents***
```
Comment (BEAdi) [inline Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md:18]:
```suggestion
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html body when provided without padding:
- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 

##### O365 Outlook Mail (Using Graph API)

Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the html body when provided without padding:
- ***send-mail***
- ***reply-mail***
- ***msgraph-mail-get-attachment***
- ***fetch-incidents*** 
```

**Review by tcarmeli1 — APPROVED** [review]
Amazing work!

**Pull request merged**